dbt/seeds/etl_full_row_counts.csv

```diff
@@ -2481,14 +2481,14 @@ out_eia__yearly_plant_parts,2014,198298
 out_eia__yearly_plant_parts,2015,204185
 out_eia__yearly_plant_parts,2016,216368
 out_eia__yearly_plant_parts,2017,223012
-out_eia__yearly_plant_parts,2018,236379
-out_eia__yearly_plant_parts,2019,249881
-out_eia__yearly_plant_parts,2020,262107
-out_eia__yearly_plant_parts,2021,274239
-out_eia__yearly_plant_parts,2022,286098
-out_eia__yearly_plant_parts,2023,298176
-out_eia__yearly_plant_parts,2024,311673
-out_eia__yearly_plant_parts,2025,301715
+out_eia__yearly_plant_parts,2018,236381
+out_eia__yearly_plant_parts,2019,249885
+out_eia__yearly_plant_parts,2020,262115
+out_eia__yearly_plant_parts,2021,274261
+out_eia__yearly_plant_parts,2022,286160
+out_eia__yearly_plant_parts,2023,298310
+out_eia__yearly_plant_parts,2024,311883
+out_eia__yearly_plant_parts,2025,303501
```

Comment:
it might be good to write down somewhere why this is happening... if i understand correctly this ppe table is influenced by the presence of new boiler generator assn and unit ids. im a lil confused why it only goes back through 2018 though i thought 2015 was the change over year for bga's...